picard/tagger.py

```diff
@@ -287,9 +301,9 @@ def _run_init(self):
                     self.add_directory(decode_filename(file))
                 else:
                     files.append(decode_filename(file))
-            if files:
-                self.add_files(files)
             del self._cmdline_files
+            if files:
+                self.add_files([files])
```

Comment:
This has been fixed. I renamed this to _add_files and created a stub `tagger.add_files()` which provides the same interface and functionality as before.